modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -13,183 +15,96 @@ $ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
-| `8` or `13`
-| No
-| -
-
-| `--icmp_code`
-| ICMP code to match on the flow.
-| `0` or `1`
-| No
-| - 	
-
-| `--peer_ip`
-| Peer IP to match on the flow.
-| `1.1.1.1` or `1::1`
-| No 	
-| -
+| Option | Description | Default
+|`--enable_pktdrop`|  enable packet drop                  | false
+|`--enable_dns`|      enable DNS tracking                 | false
+|`--enable_rtt`|      enable RTT tracking                 | false
+|`--enable_filter`|   enable flow filter                  | false
+|`--log-level`|       components logs                     | info
+|`--max-time`|        maximum capture time                | 5m
+|`--max-bytes`|       maximum capture bytes               | 50000000 = 50MB
+|`--copy`|            copy the output files locally       | prompt
+|`--direction`|       flow filter direction               | n/a
+|`--cidr`|            flow filter CIDR                    | 0.0.0.0/0
+|`--protocol`|        flow filter protocol                | n/a
+|`--sport`|           flow filter source port             | n/a
+|`--dport`|           flow filter destination port        | n/a
+|`--port`|            flow filter port                    | n/a
+|`--sport_range`|     flow filter source port range       | n/a
+|`--dport_range`|     flow filter destination port range  | n/a
+|`--port_range`|      flow filter port range              | n/a
+|`--tcp_flags`|       flow filter TCP flags               | n/a
+|`--icmp_type`|       ICMP type                           | n/a
+|`--icmp_code`|       ICMP code                           | n/a
+|`--peer_ip`|         peer IP                             | n/a
+|`--action`|          flow filter action                  | Accept
+|`--interfaces`|      interfaces to monitor               | n/a
 |===
 
-[id="cli-reference-packet-capture-options_{context}"]
-== Packet capture options
+.Example running flow capture with options:
+[source,terminal]
+----
+$ oc netobserv flows --enable_pktdrop=true  --enable_rtt=true --enable_filter=true --action=Accept --cidr=0.0.0.0/0 --protocol=TCP --port=49051
+----
+== Packets capture options
 You can filter on port and protocol for packet capture data. 
 
 .`oc netobserv packets` syntax
 [source,terminal]
 ----
 $ oc netobserv packets [<option>]
 ----
-
-.Packet capture filter option
+[cols="1,1,1",options="header"]
 |===
-|Option| Description| Possible values| Mandatory| Default
-| `--filter`
-| Enable packet capture filtering.
-| `tcp`, `udp`, or `<port>` You can specify filtering options using a comma as delimeter. For example, `tcp,80` specifies the `tcp` protocol and port `80`.
-| Yes
-| -
+| Option | Description | Default
+|`--log-level`|       components logs                     | info
+|`--max-time`|        maximum capture time                | 5m
+|`--max-bytes`|       maximum capture bytes               | 50000000 = 50MB
+|`--copy`|            copy the output files locally       | prompt
+|`--directio`n|       flow filter direction               | n/a
```

Comment:
direction word error

Reply:
I went ahead and addressed this because I added the backticks after I copied the asciidoc regeneration from the upstream doc. But I think for some of these other ones, Julien will need to update there too.